index.bs

```diff
@@ -2117,7 +2117,49 @@ If one were to write the <code>ReadableByteStreamController</code> class in some
 
 <h4 id="rbs-controller-internal-slots">Internal Slots</h4>
 
-TBA
+Instances of <code>ReadableByteStreamController</code> are created with the internal slots described in the following table:
+
+<table>
+  <thead>
+    <tr>
+      <th>Internal Slot</th>
+      <th>Description (<em>non-normative</em>)</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>\[[controlledReadableByteStream]]
+    <td>The <a href="#rbs-class"><code>ReadableByteStream</code></a> instance controlled
+  </tr>
+  <tr>
+    <td>\[[underlyingByteSource]]
+    <td>An object representation of the stream's underlying byte source
+  </tr>
+  <tr>
+    <td>\[[pullAgain]]
```

Comment:
Got it! Regarding `pulling`, I made a small change (removing fulfill) since the current pull operation of the readable byte stream doesn't use a promise.